libs/withdraws/src/lib/use-withdrawals.ts

```diff
@@ -85,41 +98,63 @@ export const useWithdrawals = () => {
     };
   }, [keypair?.pub, subscribeToMore]);
 
-  return queryResult;
+  const withdrawals = useMemo(() => {
+    return orderBy(
+      compact(data?.party?.withdrawalsConnection.edges).map(
+        (edge) => edge.node
+      ),
+      (w) => new Date(w.createdTimestamp).getTime(),
```

Comment:
`'createdTimestamp'` should be enough. ISO timestamp is safe for string comparsion